bundles/org.openhab.binding.telegram/README.md

```diff
@@ -62,29 +62,34 @@ In order to send a message, an action must be used instead.
 | `proxyPort`             |  None   | No       | Proxy port for telegram binding.                                                             |
 | `proxyType`             |  SOCKS5 | No       | Type of proxy server for telegram binding (SOCKS5 or HTTP). Default: SOCKS5                  |
 
+By default chat ids are bi-directionally, i.e. they can send and receive messages.
+They can be prefixed with an access modifier:
+- `<` restricts the chat to send only, i.e. this chat id can send messages to openHAB, but will never receive a notification.
+- `>` restricts the chat to receive only, i.e. this chat id will receive all notifications, but messages from this chat id will be discarded. 
+To use the reply function, chat ids need to be bi-directional.
 
 telegram.thing (no proxy):
 
 ```
-Thing telegram:telegramBot:Telegram_Bot [ chatIds="< ID >", botToken="< TOKEN >" ]
+Thing telegram:telegramBot:Telegram_Bot [ chatIds="ID", botToken="TOKEN" ]
 ```
 
 telegram.thing (multiple chat ids and markdown format):
 
 ```
-Thing telegram:telegramBot:Telegram_Bot [ chatIds="< ID1 >","< ID2 >", botToken="< TOKEN >", parseMode ="Markdown" ]
+Thing telegram:telegramBot:Telegram_Bot [ chatIds="ID1","ID2", botToken="TOKEN", parseMode ="Markdown" ]
 ```
```

Comment:
I guess you removed the tags here to not confuse the user with the send/receive characters. Could we modify this example to also show how it would look with the prefix e.g. use ">ID2" and to update the text in parenthesis ("... where ID2 can only receive messages"? Then we would also avoid confusions whether a space is required or not.